index.css

```
@@ -0,0 +1,287 @@
+@font-face
+{
+    font-family: 'PT SansNarrow';
+    src: url('fonts/PTN57F.ttf');
+}
+
+@font-face
+{
+    font-family: Arial;
+    src: local(arial);
+}
+
+html
+{
+    height: 100%;
+}
+
+body
+{
+    background-color: antiquewhite;
+    margin: 0;
+    height: 100%;
+    overflow-x: hidden;
+    position: relative;
+}
+
+.cite-name
```

Может имелось в виду
```suggestion
.site-name
```
?